app/screens/create/create_challenge.tsx

```diff
@@ -40,8 +36,12 @@ const CreateChallengeScreen = ({
         date,
         description,
         currentLocation,
+        image_id,
```

Comment:
The createChallenge function in ChallengeBuilder should make image_id mandatory to create the challenge for all the challenges to post the image_id and retrieve them

Reply:
For now we should leave it like this bc we have to also think about the location etc...

Reply:
I don't understand your point. What is the link with location ?
The location is either set or kept "null" but it is mandatory too (on the PR #189)